CHANGELOG.md

```diff
@@ -11,4 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - A changelog file.
 
+### Fixed
+
+- Make monero_wallet_rpc readiness check language agnostic. The readiness check was
+  failing on non-english language systems preventing users from starting the swap-cli
+  and asb.
```

Comment:
I'd prefer if we stick to the one-sentence-per-line rule for text source files like this.
Git works on a line-by-line basis and in natural language, one usually changes full sentences.

Reply:
We will have to configure dprint to accept it then. Doesn't pass CI unless I `dprint fmt`

Reply:
Oh really? :(

My beloved dprint is wrapping markdown files at a certain character range? Damn.

Reply:
I guess we could just set the line length to 200 chars or something: https://dprint.dev/plugins/markdown/config/